contrib/src/openssl/rules.mak

```diff
@@ -107,12 +107,27 @@ export CROSS_SDK=iPhone${IOS_PLATFORM}.sdk
 endif
 
 ifdef HAVE_TVOS
+
 ifeq ($(MY_TARGET_ARCH),arm64)
-OPENSSL_CONFIG_VARS=ios64-cross
```

Comment:
The compilation for iOS remains unchanged a little higher on the file. This code only affects tvOS, as it was under `ifdef HAVE_TVOS` but specified the wrong configs (`ios64-cross` and `darwin64-x86_64-cc`). The `ios64-cross` is [still used for iOS](https://github.com/pyrosphere/cocos2d-x-3rd-party-libs-src/blob/9add252849d74be334fbabaaa9c8ce3ebdd07730/contrib/src/openssl/rules.mak#L77).